README.md

```diff
@@ -15,6 +15,16 @@ twinkles1_and_index
 
 Astrometry index catalogs corresponding to the twinkles1 data.
 
+cfht
+----
+
+Source catalogs and metadata from the validation_data_cfht repository. The calexps were processed via the included `compress_jointcal_cfht_test_data.py` to remove the pixel-level data and gzip compress them to make their size reasonable. I also made a skyMap and added it to deepCoadds.
```

Comment:
So this package is intended as a common and convenient test data set, rather than for regression testing?  I'm concerned that including processed data products (e.g., calexps, source catalogs) means that they won't track changes and regressions in other products (e.g., processCcd.py in pipe_tasks).  But if it's clear that this is not intended for regression testing then that's fine; but we surely need something else for integration and regression testing.